vnext/ReactWindowsCore/ReactWindowsCore.vcxproj

```diff
@@ -131,6 +131,7 @@
     <ClInclude Include="NativeModuleProvider.h" />
     <ClInclude Include="OInstance.h" />
     <ClInclude Include="pch.h" />
+    <ClInclude Include="PlatformConstantsModule.h" />
```

Comment:
>"P [](start = 23, length = 2)

Why is it not in the Modules folder?

Reply:
Thanks for catching this! I had thought Visual Studio was a bit more automatic updating these files. Will fix the things that I moved.